data_validation/partition_builder.py

```diff
@@ -316,43 +340,39 @@ def _add_partition_filters(
         ConfigManager objects.
 
         Args:
-            partition_filters (List[List[List[str]]]): List of List of Partition filters
-            for all Table/ConfigManager objects, two list of filters for each configManager object,
-            one for the source and one for the target
-
+            self.config_managers is a list of table pairs to which the partition filters must be applied.
```

Comment:
config_managers isn't a list of table pairs.. are your referring to the fact that each ConfigManager represents a single table validation?